src/routes/Search.js

```diff
@@ -2,27 +2,33 @@ import React from 'react';
 import { RouteHandler } from 'react-router';
 import axios from 'axios';
 import FlexView from 'react-flexview';
+import LoadingSpinner from '../components/LoadingSpinner';
```

Comment:
just FYI, you can import `from 'components/LoadingSpinner` (like other imports below) because webpack was configured to do so. Relative imports are always fine anyway